src/components/RoleField.jsx

```diff
@@ -38,15 +46,6 @@ const RoleField = (props) => {
   );
 };
 
-RoleField.propTypes = {
-  // The name of the Input
-  name: PropTypes.string.isRequired,
-  // The role that is currently selected
-  currentValue: PropTypes.string,
-  // A change handler for when the user is picking their role
-  onNewValue: PropTypes.func.isRequired,
-};
-
 RoleField.defaultProps = {
```

Comment:
This is the weird thing with React. You actually can't move it into the class since it is not a class. It's a pure function. So it has to stay outside